src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -186,11 +186,12 @@ public interface DsVersionHandler<T> {
     @GET
     @AuthRequired
     @Path("{id}")
-    public Response getDataset(@Context ContainerRequestContext crc, @PathParam("id") String id, @Context UriInfo uriInfo, @Context HttpHeaders headers, @Context HttpServletResponse response) {
+    public Response getDataset(@Context ContainerRequestContext crc, @PathParam("id") String id, @Context UriInfo uriInfo, @Context HttpHeaders headers, @Context HttpServletResponse response,  @QueryParam("returnOwners") Boolean returnOwners) {
         return response( req -> {
             final Dataset retrieved = execCommand(new GetDatasetCommand(req, findDatasetOrDie(id)));
             final DatasetVersion latest = execCommand(new GetLatestAccessibleDatasetVersionCommand(req, retrieved));
-            final JsonObjectBuilder jsonbuilder = json(retrieved);
+            Boolean includeOwners = returnOwners == null ? false : returnOwners;
```

Comment:
You can directly use the primitive boolean type: `@QueryParam("returnOwners") boolean returnOwners` 

And thus avoid the null check. If the query param is not set in the URL, `false` is the default value.